.github/workflows/main.yml

```
@@ -0,0 +1,77 @@
+name: Generate new docs
+
+on:
+  workflow_dispatch:
+    inputs:
+      project:
+        description: 'ember or ember-data'     
+        required: true
+      version:
+        description: 'The version to build, i.e. 3.20.0'     
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Clone the project repo
+        run: |
+          set -euo pipefail
+          if [$INPUT_PROJECT -eq 'ember']
+          then
+            git clone https://github.com/emberjs/ember.js
+            cd ember.js
+            git checkout "v$INPUT_VERSION"
+          elif [$INPUT_PROJECT -eq 'ember-data']
+          then
+            git clone https://github.com/emberjs/data
+            cd data
+            git checkout "v$INPUT_VERSION"
+          else
+            echo "Please specify either ember or ember-data for the project."
+          fi
+
+      - name: Clone ember-api-docs-data
+        run: git clone https://github.com/ember-learn/ember-api-docs-data
+      
+      - name: yarn install
+        run: yarn install
+
+      - name: Generate s3-docs
+        working-directory: ember-jsonapi-docs
+        run: yarn gen --project $INPUT_PROJECT --version $INPUT_VERSION
+
+      - name: Generate json-docs
+        working-directory: ember-jsonapi-docs
+        run: yarn start --project $INPUT_PROJECT --version $INPUT_VERSION
```

```suggestion
        run: yarn start --project "${{ github.event.inputs.project }}" --version "${{ github.event.inputs.version }}"
```

Same caveat as above